### PR TITLE
WebtoonXYZ fix 

### PR DESCRIPTION
**Pull request by MareGraphics**
Fixed unused const which broke the presence.
I hope I fixed my fork, had me so angered..

### CODE REVIEW

File: websites/W/WebtoonXYZ/presence.ts [hunk 1]
```diff
@@ -65,7 +64,6 @@ presence.on("UpdateData", async () => {
       data.buttons = [{ label: "Read Webtoon", url: window.location.href }];
   } else if (pathname.startsWith("/read")) {
     const title = document.querySelector(".post-title").textContent;
-    data.details = "Viewing:";
```
Comment (ririxidev) [inline websites/W/WebtoonXYZ/presence.ts:68]: Why

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/W/WebtoonXYZ/presence.ts (hunk 1 above)
Comment (MareGraphics) [reply websites/W/WebtoonXYZ/presence.ts:68]: oooh oops, i had a console log there so I probably clicked CTRL SHIFT K 2 times

File: websites/W/WebtoonXYZ/dist/metadata.json [hunk 2]
```diff
@@ -38,4 +38,4 @@
       ]
     }
   ]
-}
\ No newline at end of file
+}
```
Comment (ririxidev) [inline websites/W/WebtoonXYZ/dist/metadata.json]: revert these changes lul

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/W/WebtoonXYZ/dist/metadata.json (hunk 2 above)
Comment (MareGraphics) [reply websites/W/WebtoonXYZ/dist/metadata.json]: Done

**Review by ririxidev — APPROVED** [review]

**Comment by ririxidev** [comment]
i hope it will not break anything - if something will revert

**Comment by MareGraphics** [comment]
I hope too..

**Review by Bas950 — APPROVED** [review]
